conf/documentation.conf

```diff
@@ -1507,12 +1507,6 @@ description=<<EOT
 Interval at which PacketFence runs person cleanup
 EOT
 
-[maintenance.online_node_count_interval]
```

Comment:
Seems this job wasn't migrated to the new pfmon framework ?

Is there something else that replaces it ? Or is it just deprecated ?